data/35e/wizards_of_the_coast/supplement/weapons_of_legacy/wl_abilities.lst

```diff
@@ -1,10 +1,275 @@
-Bonus Legacy Feat			KEY:Legacy Champion ~ Bonus Legacy Feat		CATEGORY:Special Ability	TYPE:Class Features.Legacy Champion Class Feature.SpecialQuality				DESC:At 4th and 8th level, you gain a bonus feat related to your legacy item, which can be selected from any of the legacy feats in this chapter for which you meet the prerequisites.																																																																																																																														SOURCEPAGE:p.20
+Legacy Champion Class Level				KEY:Legacy Champion ~ Class Level						CATEGORY:Special Ability	TYPE:Class Features.Legacy Champion Class Feature.SpecialQuality					DESC:At level 1 and 7 advance the Legacy Champion class, on level 2-6 and 8-10 advance classes in which you had levels before entering the Legacy Champion class. After each levelup, select this stacking ability to increase your Legacy Champion level. 																																																																																																																																																							SOURCEPAGE:p.19																								VISIBLE:DISPLAY		COST:0				BONUS:VAR|LegacyChampionLVL|1						PRECLASS:1,Legacy Champion=1		CHOOSE:NUMCHOICES=10|NOCHOICE	MULT:YES	STACK:YES									BONUS:ABILITYPOOL|Legacy Champion Stat Adjustment|1
+Class Features							KEY:Legacy Champion ~ Class Features					CATEGORY:Special Ability	TYPE:Class Features.Legacy Champion Class Feature.SpecialQuality					DESC:At each level except 1st and 7th, you gain class features and an increase in effective level as if you had also gained a level in a class to which you belonged before adding the prestige class level. The specific class features you gain include spells per day (and spells known, if applicable), improved chance of turning or destroying undead, metamagic or item creation feats, bonus feats, monk special abilities, sneak attack progressions, and so on, depending on the class. You do not, however, gain the benefit of your previous class's Hit Dice, attack progression, skill points, or saving throws. If you had more than one class before becoming a legacy champion, you must decide to which class to add each level for the purpose of determining class abilities. Your effective Legacy Champion class level is %1.|LegacyChampionLVL				SOURCEPAGE:p.19		BONUS:ABILITYPOOL|Legacy Champion Class Feature|1|PREVARGTEQ:LegacyChampionLVL,1							COST:100			BONUS:ABILITYPOOL|Legacy Champion Class Feature|1|PREVARGTEQ:LegacyChampionLVL,2		BONUS:ABILITYPOOL|Legacy Champion Class Feature|1|PREVARGTEQ:LegacyChampionLVL,3		BONUS:ABILITYPOOL|Legacy Champion Class Feature|1|PREVARGTEQ:LegacyChampionLVL,5		BONUS:ABILITYPOOL|Legacy Champion Class Feature|1|PREVARGTEQ:LegacyChampionLVL,6		BONUS:ABILITYPOOL|Legacy Champion Class Feature|1|PREVARGTEQ:LegacyChampionLVL,9		BONUS:ABILITYPOOL|Legacy Champion Class Feature|1|PREVARGTEQ:LegacyChampionLVL,10
```

Comment:
This might be better represented as DEFINE:LegacyChampionClassFeatureCount|0
BONUS:VAR|LegacyChampionClassFeatureCount|(LegacyChampionLVL>=2)+(LegacyChampionLVL>=3)+(LegacyChampionLVL>=5)+(LegacyChampionLVL>=6)+(LegacyChampionLVL>=9)+(LegacyChampionLVL>=10)

BONUS:ABILITYPOOL|Legacy Champion Class Feature|LegacyChampionClassFeatureCount

That reduces the number of calculation cycles against a prevar.